### PR TITLE
[16.0][OU-ADD] web: Nothing to do

### PR DESCRIPTION
**Pull request by marielejeune**
Nothing to do

### CODE REVIEW

**Comment by marielejeune** [comment]
https://github.com/OCA/OpenUpgrade/issues/3681

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
- Please rebase the branch over latest 16.0 to get full tests working.
- You have to put `Done` or `Nothing to do` in this file: https://github.com/OCA/OpenUpgrade/blob/16.0/docsource/modules150-160.rst for activating the CI for that module (and also for documentation purposes).
- The commit message should be `[OU-ADD] web: Nothing to do`, and don't put the version on it as prefix (this can apply to entire OCA).

Please do the same for the rest of your pull requests.

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -790,7 +790,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | utm                                             |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| web                                             |                      |                                                 |
+| web                                             |Nothing to do         |                                                 |
```
Comment (pedrobaeza) [inline docsource/modules150-160.rst]: We leave by convention an space before (as you have done also in other PRs).

```suggestion
| web                                             | Nothing to do        |                                                 |
```

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**